aplib.net-demo/Assets/Scripts/Enemy/DummyEnemy.cs

```diff
@@ -0,0 +1,36 @@
+using UnityEngine;
+
+/// <summary>
+/// Dummy basic enemy class to use as target during weapons testing.
+/// It respawns after dying.
+/// </summary>
+public class DummyEnemy : BasicEnemy
+{
+    /// <summary>
+    /// The spawn area for the enemy. 
+    /// The size is defined by the local scale of the transform.
+    /// </summary>
+    [SerializeField] private Transform _spawnArea;
```

Comment:
This would need some change if you would actually use some Area object, but would be a lot cleaner.